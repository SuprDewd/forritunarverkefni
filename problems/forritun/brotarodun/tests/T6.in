4
645/944
943/917
761/325
529/750
