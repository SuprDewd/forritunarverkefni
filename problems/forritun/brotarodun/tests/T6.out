645/944
529/750
943/917
761/325
